src/webhook/handlers/askingArticleSubmissionConsent.js

```diff
@@ -53,6 +55,8 @@ export default async function askingArticleSubmissionConsent(params) {
       }
     `({ text: data.searchedText }, { userId });
 
+    await UserArticleLink.create({ userId, articleId: CreateArticle.id });
```

Comment:
As discussed in slack today, we should use an API that also writes `lastViewedAt`.

Also, please add `lastViewedAt` as a required field in `userArticleLink.json` to ensure that `lastViewedAt` always exists in DB.